appveyor.yml

```diff
@@ -13,32 +13,38 @@ environment:
     - PYTHON: "C:\\Python27"
       PYTHON_VERSION: "2.7.11"
       PYTHON_ARCH: "32"
+      NOX_ENV: "interpreters(version='2.7')"
```

Comment:
Nox uses 'session' instead of 'env', so let's keep it consistent and use `NOX_SESSION` as the env var.